LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 RStudio
+Copyright (c) 2023 RStudio
```

Comment:
Is this an appropriate time to run the `s/RStudio/Posit/g` substitution?

Reply:
Yes! I missed!